src/commands/shared.ts

```diff
@@ -187,10 +187,19 @@ export const provisionRequest = async (
     throw "Public key mismatch. Please revoke the request and try again.";
   }
 
-  return { request: provisionedRequest, publicKey, privateKey };
+  const posixAccount =
+    provisionedRequest.permission.spec.type === "gcloud"
```

Comment:
and everything google-related into `plugins/google`

Reply:
`importSshKey` function is in `plugins/google`.
This file is shared between scp and ssh not between clouds.
If we don't want to have branching like this I think we have to couple data with implementation, which we don't do now in general.